dora-parser/src/ast.rs

```diff
@@ -602,9 +637,11 @@ impl Function {
     }
 }
 
+//remove in next step
```

Comment:
IMHO it's not necessary to add these comments, but if you do please add a space after the slashes